transports/webrtc/src/tokio/udp_mux.rs

```diff
@@ -398,11 +398,19 @@ impl UDPMuxNewAddr {
 
                             continue;
                         }
-                        Poll::Ready(Err(err)) if err.kind() == ErrorKind::TimedOut => {}
```

Comment:
I'd prefer if we stick to this pattern matching style, it results in a flatter code hierarchy.

Reply:
Yes agreed, I'll reformat it

Reply:
I made these flatten changes, marked this as resolved, yet Github still shows as outstanding. I'm a bit confused as to why...

Reply:
As far as I can tell, this is due to the line above, namely the original line, not having changed. I think we are all good here. Thanks @DougAnderson444 for circling back on it.